Common/cpp/headers/LayoutAnimations/LayoutAnimationsProxy.h

```diff
@@ -0,0 +1,33 @@
+#ifndef LayoutAnimationsProxy_h
+#define LayoutAnimationsProxy_h
```

Comment:
Why you use `#ifdef` instead of `#pragma once`? I noticed this in a couple of places.